libraries/debug-certificate-manager/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@microsoft/debug-certificate-manager",
```

Comment:
```suggestion
  "name": "@rushstack/debug-certificate-manager",
```

Reply:
I was talking to APC about this. Unless we plan on moving over to the new group name in the next week or so, I think it would be best to leave it as @microsoft for now and let it be part of the mass update later. How do you feel about that?

Reply:
Let's just start out with the right scope so we don't have to potentially tombstone the @microsoft package.